tests/helics/core/MpiCore-tests.cpp

```diff
@@ -12,7 +12,8 @@ Lawrence Livermore National Laboratory, operated by Lawrence Livermore National
 #include <boost/test/unit_test.hpp>
 
 #if HELICS_HAVE_MPI
```

Comment:
I think I would probably just put the conditional in the Cmake so it only adds the file if MPI is present